_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kubeflow 1.10 Release Announcement"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.10-release/
+author: "Kubeflow 1.10 Release Team, Dimitris Poulopoulos"
+---
+
+Kubeflow 1.10.0 delivers essential updates that enhance the flexibility, efficiency, and scalability of machine learning
+workflows. The new features span across several components, improving both user experience and system performance.
+
+## Benefits to the ecosystem
+
+* Migration to GitHub Container Registry (ghcr) as default container registry for (most) images
+
+## Highlight features
+
+* Trainer 2.0
+* Model Registry UI
+* Spark Operator as a core Kubeflow component
+* Kubernetes and container security (CISO compatibility)
+* LLM hyper parameter tuning
+
+The upcoming Kubeflow release (v1.10) is packed with exciting new features and enhancements designed to streamline your
+machine learning workflows and improve security. Key highlights include the introduction of a Model Registry UI, the new
+Kubeflow Trainer 2.0, and the integration of the Spark Operator as a core Kubeflow component. This release also places a
+strong emphasis on Kubernetes and container security, addressing critical CISO compatibility requirements.
+
+Beyond these additions, Kubeflow v1.10 delivers significant improvements across the platform. Pipelines gain flexibility
+with loop parallelism, enabling more dynamic and efficient workflows. Katib expands its hyperparameter optimization
+capabilities with new parameter distributions and a high-level API tailored for fine-tuning large language models
+(LLMs). Model Registry provides deeper integrations with KServe, while KServe itself brings a new Python SDK, OCI
+storage for models, and model caching, boosting model deployment speed and reliability. Finally, the Spark operator
+enhancements deliver secure and scalable cluster management with new security contexts and rootless Istio-CNI
+integrations.
+
+## Kubeflow Platform (Manifests & Security)
+
```

Comment:
We might want an intro sentence to the Manifests and Security Sections....something shorter than this...     The Kubeflow Platform Working Group is committed to simplifying Kubeflow's installation and operations.   We continue to update and test the installation manifests of Kubeflow's integrated applications and their dependencies.   In addition, we are investing in improving our security profile.   Please find details on our work below: